Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -219,11 +260,25 @@ namespace ROS2
         // TODO - solution: once Collider Component supports Cylinder Shape, switch to it from Shape Collider Component.
 
         AZ::Entity* entity = AzToolsFramework::GetEntityById(entityId);
+        AZ_Assert(entity, "entity not exists");
         auto geometry = collision->geometry;
         bool isPrimitiveShape = geometry->type != urdf::Geometry::MESH;
+
+        Physics::ColliderConfiguration colliderConfig;
+        const bool is_wheel = GuessIsWheel(entity->GetName());
+        if (is_wheel)
+        {
+            colliderConfig.m_materialSlots.SetMaterialAsset(0, m_wheelMaterial);
+        }
+        colliderConfig.m_position = URDF::TypeConversions::ConvertVector3(collision->origin.position);
+        if (!isPrimitiveShape)
+            colliderConfig.m_rotation = URDF::TypeConversions::ConvertQuaternion(collision->origin.rotation);
+
         if (!isPrimitiveShape)
         {
-            entity->CreateComponent<PhysX::EditorColliderComponent>();
```

Comment:
move code to subroutine. 
here, check isWheelEntity, warn that this case is not supported yet